### PR TITLE
Docs: add FAQs

### PR DESCRIPTION
**Pull request by mahjonp**
Signed-off-by: mahjonp <junpeng.man@gmail.com>

### What problem does this PR solve? 

Add a FAQs document, refer: #151 

### Check List 

```
None
```

Code changes

```
None
```

Side effects

```
None
```

Related changes

```
None
```

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: doc/faqs.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+### Q: chaosfs sidecar container run failed, and log shows `pid file found, ensure docker is not running or delete /tmp/fuse/pid`
+
+```
+2020-01-19T06:30:56.629Z	INFO	chaos-daemon	Init hookfs
+2020-01-19T06:30:56.630Z	ERROR	chaos-daemon	failed to create pid file	{"error": "pid file found, ensure docker is not running or delete /tmp/fuse/pid"}
+github.com/go-logr/zapr.(*zapLogger).Error
+```
+
+* Cause: Chaos-mesh uses fuse to hijack I/O operations, it will fail if there already exists the directory which chosen as the source path. This often happens when reuses a `Retain` reclaim policy PV when request a PVC resource.
```
Comment (dcalvin) [inline doc/faqs.md]:
```suggestion
* Cause: Chaos-mesh uses fuse to hijack I/O operations. It fails if you specify an existing directory as the source path for chaos. This often happens when you try to reuse a persistent volume (PV) with the `Retain` reclaim policy to request a PVC resource.
```

File: doc/faqs.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+* Solution: In this case, you could use below command to change the reclaim policy to `Delete`:
```
Comment (dcalvin) [inline doc/faqs.md]:
```suggestion
* Solution: In this case, use the following command to change the reclaim policy to `Delete`:
```

File: doc/faqs.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+### Q: chaosfs sidecar container run failed, and log shows `pid file found, ensure docker is not running or delete /tmp/fuse/pid`
+
+```
+2020-01-19T06:30:56.629Z	INFO	chaos-daemon	Init hookfs
+2020-01-19T06:30:56.630Z	ERROR	chaos-daemon	failed to create pid file	{"error": "pid file found, ensure docker is not running or delete /tmp/fuse/pid"}
+github.com/go-logr/zapr.(*zapLogger).Error
+```
+
+* Cause: Chaos-mesh uses fuse to hijack I/O operations. It fails if you specify an existing directory as the source path for chaos. This often happens when you try to reuse a persistent volume (PV) with the `Retain` reclaim policy to request a PVC resource.
```
Comment (cwen0) [inline doc/faqs.md]: The Sidecar container will restarted all the time. You can describe what happens in this case.

File: doc/faqs.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+The chaosfs sidecar container is continuously restarting, and you may see following logs at the current sidecar container:
+
+```
+2020-01-19T06:30:56.629Z	INFO	chaos-daemon	Init hookfs
+2020-01-19T06:30:56.630Z	ERROR	chaos-daemon	failed to create pid file	{"error": "pid file found, ensure docker is not running or delete /tmp/fuse/pid"}
+github.com/go-logr/zapr.(*zapLogger).Error
+```
+
+* Cause: Chaos-mesh uses fuse to hijack I/O operations. It fails if you specify an existing directory as the source path for chaos. This often happens when you try to reuse a persistent volume (PV) with the `Retain` reclaim policy to request a PVC resource.
```
Comment (dcalvin) [inline doc/faqs.md]:
```suggestion
* Cause: Chaos-mesh uses fuse to hijack I/O operations. It fails if you specify an existing directory as the source path for chaos. This often happens when you try to reuse a persistent volume (PV) with the `Retain` reclaim policy to request a PersistentVolumeClaims (PVC) resource.
```

File: doc/faqs.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+* Solution: In this case, use the following command to change the reclaim policy to `Delete`:
```
Comment (dcalvin) [inline doc/faqs.md]:
```suggestion
* **Solution**: In this case, use the following command to change the reclaim policy to `Delete`:
```

File: README.md [hunk 6]
```diff
@@ -222,6 +222,10 @@ kubectl port-forward -n chaos-testing svc/chaos-dashboard 8080:80
 
 Then you can access [`http://localhost:8080`](http://localhost:8080) in browser.
 
+## FAQs
+
+refer to: [doc/FAQs](/doc/faqs.md)
```
Comment (dcalvin) [inline README.md]:
```suggestion
Refer to [FAQs](/doc/faqs.md).
```

**Pull request merged**